src/App.tsx

```diff
@@ -14,12 +14,12 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
+import Backdrop from '@material-ui/core/Backdrop';
+import CircularProgress from '@material-ui/core/CircularProgress';
```

Comment:
You can one-line these 2 by:
```
import { CircularProgress,  Backdrop } from '@material-ui/core';
```

Reply:
yes, but by doing we would be importing the whole material package and the bundle size would increase. With this such a big packages i prefer to be explicit and bring the specific packages. I know that some configuration can be done as far i could read but i like being explicit on this cases ( same with lodash )